crates/basilica-sdk-python/examples/ssh_utils.py

```
@@ -0,0 +1,121 @@
+    
+    Args:
+        credentials: SSH credentials string (e.g., 'root@84.200.81.243:32776')
+        
+    Returns:
+        Tuple of (user, host, port)
+        
+    Raises:
+        ValueError: If credentials format is invalid
+    """
+    if not credentials:
+        raise ValueError("SSH credentials cannot be empty")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)